.github/workflows/_rust.yml

```diff
@@ -110,6 +110,8 @@ jobs:
         run: cargo build -p firezone-windows-client
       - name: Run smoke test
         run: cargo run -p firezone-windows-client -- smoke-test
+      - name: Test that the crash handler produces a crash dump
+        run: bash scripts/crash-handling-smoke-test.bash
```

Comment:
Should we just set the default shell to be bash?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-set-the-default-shell-and-working-directory

Reply:
Good idea, I'll use that next time this changes, since the Windows stuff will gradually be changed to Bash.